src/utils_format_kairosdb.c

```diff
@@ -181,7 +181,8 @@ static int values_to_kairosdb(char *buffer, size_t buffer_size, /* {{{ */
 
 static int value_list_to_kairosdb(char *buffer, size_t buffer_size, /* {{{ */
                                   const data_set_t *ds, const value_list_t *vl,
-                                  int store_rates) {
+                                  int store_rates, char **http_attrs,
+                                  int http_attrs_num) {
```

Comment:
You're mixing data types, here http_attrs_num is an int, but it is declared as a size_t in write_http.c